plugin/route53/setup_test.go

```diff
@@ -68,9 +69,11 @@ func TestSetupRoute53(t *testing.T) {
 	}
 
 	for _, test := range tests {
-		c := caddy.NewTestController("dns", test.body)
-		if err := setup(c, f); (err == nil) == test.expectedError {
-			t.Errorf("Unexpected errors: %v", err)
-		}
+		t.Run(fmt.Sprintf("Test case: '%s'", test.body), func(t *testing.T) {
```

Comment:
Why not just print the test number that you can get from the range loop? We do that in a lot of other tests; that's a 2 line change

Reply:
Printing the test case itself allows you to see immediately what the error is. This is particularly useful when a change triggers failure in multiple test cases, printing the test case allows you to see immediately the failure pattern.

IMO, this is a big time saver.